README.md

```diff
@@ -1,6 +1,6 @@
-# asdf-ephemeral-pg ![Build](https://github.com/smashedtoatoms/asdf-ephemeral-postgres/workflows/Build/badge.svg?branch=master)
+# asdf-ephemeral-postgres ![Build](https://github.com/smashedtoatoms/asdf-ephemeral-postgres/workflows/Build/badge.svg?branch=master)
```

Comment:
I can't really change the name of the plugin without breaking things for folks.  Any chance you can update this to not change the name?  If it's confusing we'll have to address it in the readme.

Reply:
`asdf plugin list all` doesn't list any "ephemeral" plugins, and the list at https://github.com/asdf-vm/asdf-plugins don't list such name either. So where this name is used?

Reply:
Oh, haha, I guess I never added it to the official list.  In that case, if we rename it now, Github should forward it for anyone that installed it using the original project name.  In that case, I will accept the change and rename the repo.  If it causes you any grief, you'll probably need to reinstall the plugin.

Reply:
Nope, I was right.  It is in the official list.  It was added before they added the helpers to do the auto-installs though.  As it is, it will keep working, so I will leave it on the official list with the old name since GitHub will forward it correctly.  If I have time in the future, I will update the official site and add the helper for the auto-install.

Reply:
`-postgres` name is easier to discover among plugins, but I have no preference.